js_modules/dagster-ui/packages/ui-core/src/search/SearchResults.tsx

```diff
@@ -232,12 +232,16 @@ export type SearchResultsProps<T extends ResultType> = {
   onClickResult: (result: T) => void;
   queryString: string;
   results: T[];
+  searching: boolean;
 };
 
 export const SearchResults = <T extends ResultType>(props: SearchResultsProps<T>) => {
-  const {highlight, onClickResult, queryString, results} = props;
+  const {highlight, onClickResult, queryString, results, searching} = props;
 
   if (!results.length && queryString) {
+    if (searching) {
+      return;
```

Comment:
Nit: Feels a little weird not to return anything from a component. Return `null` or `<div />` or something else non-undefined?